src/libraries/kunena/src/User/KunenaBan.php

```diff
@@ -811,7 +811,8 @@ public function save($updateOnly = false): bool
 
 		if ($this->userid)
 		{
-			$user = Factory::getApplication()->getIdentity($this->userid);
```

Comment:
Yeah, the above is supposed to get the current user. The parameter is for initialization only.

Reply:
The issue is that the method getIdentity() doesn't accept any parameters